### PR TITLE
incident report for 2018-04-18

### PR DESCRIPTION
**Pull request by minrk**

### CODE REVIEW

File: docs/source/incident-reports/2018-04-18-cull-flood.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+### 17:03
+
+Believing the issue to be resolved, work resumes,
+merging new pull requests into binderhub.
+
+### 17:23
+
+Observing culler behavior is fixed, but memory and CPU growth continues.
+Reviewing further changes that were part of the revised deploy,
+[a new feature of kubespawner]() is suspected as the source of the leak.
```
Comment (betatim) [inline docs/source/incident-reports/2018-04-18-cull-flood.md]: missing link?

**Comment by betatim** [comment]
Thanks for the write up and patience in fixing this on the days.

Should we briefly discuss some (automatic/frequent deploys) of the action items from this in this months team meeting?

**Comment by minrk** [comment]
Yes, I think it's worth bringing up again, even if we come to the same conclusion. We have more experience now.

**Comment by betatim** [comment]
Merge?

**Pull request merged**